docs/maintainers-guide.md

```diff
@@ -0,0 +1,59 @@
+# Maintainers Guide
+
+This page offers guidance to project maintainers regarding our setup procedures, release processes, package creation, and other related tasks.
+
+# Maintainer Onboarding and Best Practices
+
+## Becoming a Maintainer or Triager
+
+If you are interested in becoming a maintainer, you can join our community. We expect you to be actively involved.
```

Comment:
```suggestion
If you are interested in becoming a maintainer, you can join our community. Maintainers have several important responsibilities, so please read on to understand the role.
```

I feel the "actively involved" part is more clearly explained in the next section :)